code/modules/surgery/encased.dm

```diff
@@ -3,7 +3,7 @@
 //				GENERIC	RIBCAGE SURGERY							//
 //////////////////////////////////////////////////////////////////
 /decl/surgery_step/open_encased
-	name = "Saw through bone"
+	name = "Saw Through Bone"
```

Comment:
This capitalisation is inconsistent with all other surgeries. Keep it "Saw through bone" and do the same for the rest you modified.

Reply:
surgical steps were mixed between non-titlecase and titlecase. all surgical steps now use titlecase, as per 96e0373.